pkg/kubectl/cmd/log.go

```diff
@@ -72,115 +72,144 @@ func selectContainer(pod *api.Pod, in io.Reader, out io.Writer) string {
 	}
 }
 
-type logParams struct {
-	containerName string
+type LogsOptions struct {
+	Client *client.Client
+
+	PodNamespace  string
+	PodName       string
+	ContainerName string
+	Follow        bool
+	Timestamps    bool
+	Interactive   bool
+	Previous      bool
+	LimitBytes    int
+	Tail          int
+	SinceTime     string
+	SinceSeconds  time.Duration
+
+	Out io.Writer
 }
 
 // NewCmdLog creates a new pod log command
 func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
-	params := &logParams{}
+	o := &LogsOptions{
+		Out:         out,
+		Interactive: true,
```

Comment:
Last time I checked interactive wasn't used anywhere? Mind removing it if thats' the case?